pyatlan/model/search.py

```diff
@@ -1893,8 +1905,21 @@ def validate_query(cls, v, values):
         else:
             raise ValueError("Either query or post_filter is required")
 
+    # Transforming dictionaries into SortItem instances to facilitate deserialization.
+    # This conversion occurs prior to invoking the initializer and validator_sort(),
+    # preventing potential validation-related issues from arising.
+    @validator("sort", pre=True)
+    def dict_to_SortItem(cls, sort):
```

Comment:
Since we're using `SortItem.from_dict`, I would rename the validator to `sortitem_from_dict()` for clarity:

```suggestion
    def sortitem_from_dict(cls, sort):
```